<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>985</width>
    <height>621</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Verdana</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>OpenMeta[analyst]</string>
  </property>
  <property name="windowIcon">
   <iconset resource="images/icons.qrc">
    <normaloff>:/images/meta.png</normaloff>:/images/meta.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="font">
    <font>
     <family>Verdana</family>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>ArrowCursor</cursorShape>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>2</number>
    </property>
    <property name="rightMargin">
     <number>2</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QFrame" name="nav_frame">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="spacing">
        <number>1</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>1</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QTableView" name="tableView">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="alternatingRowColors">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QFrame" name="frame">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>65</height>
            </size>
           </property>
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <widget class="QFrame" name="frame_2">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>0</y>
              <width>181</width>
              <height>61</height>
             </rect>
            </property>
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <widget class="QToolButton" name="nav_left_btn">
             <property name="geometry">
              <rect>
               <x>0</x>
               <y>10</y>
               <width>20</width>
               <height>41</height>
              </rect>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset resource="images/icons.qrc">
               <normaloff>:/images/left_arrow.png</normaloff>:/images/left_arrow.png</iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>64</width>
               <height>64</height>
              </size>
             </property>
             <property name="autoRaise">
              <bool>true</bool>
             </property>
            </widget>
            <widget class="QToolButton" name="nav_up_btn">
             <property name="geometry">
              <rect>
               <x>40</x>
               <y>0</y>
               <width>51</width>
               <height>20</height>
              </rect>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset resource="images/icons.qrc">
               <normaloff>:/images/up_arrow.png</normaloff>:/images/up_arrow.png</iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>64</width>
               <height>64</height>
              </size>
             </property>
             <property name="autoRaise">
              <bool>true</bool>
             </property>
            </widget>
            <widget class="QToolButton" name="nav_down_btn">
             <property name="geometry">
              <rect>
               <x>40</x>
               <y>40</y>
               <width>51</width>
               <height>20</height>
              </rect>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset resource="images/icons.qrc">
               <normaloff>:/images/down_arrow.png</normaloff>:/images/down_arrow.png</iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>64</width>
               <height>64</height>
              </size>
             </property>
             <property name="toolButtonStyle">
              <enum>Qt::ToolButtonIconOnly</enum>
             </property>
             <property name="autoRaise">
              <bool>true</bool>
             </property>
             <property name="arrowType">
              <enum>Qt::NoArrow</enum>
             </property>
            </widget>
            <widget class="QLabel" name="nav_lbl">
             <property name="geometry">
              <rect>
               <x>20</x>
               <y>20</y>
               <width>91</width>
               <height>21</height>
              </rect>
             </property>
             <property name="font">
              <font>
               <pointsize>10</pointsize>
               <weight>50</weight>
               <bold>false</bold>
              </font>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
            <widget class="QToolButton" name="nav_right_btn">
             <property name="geometry">
              <rect>
               <x>110</x>
               <y>10</y>
               <width>20</width>
               <height>41</height>
              </rect>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset resource="images/icons.qrc">
               <normaloff>:/images/right_arrow.png</normaloff>:/images/right_arrow.png</iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>64</width>
               <height>64</height>
              </size>
             </property>
             <property name="autoRaise">
              <bool>true</bool>
             </property>
            </widget>
            <widget class="QToolButton" name="nav_add_btn">
             <property name="geometry">
              <rect>
               <x>130</x>
               <y>10</y>
               <width>51</width>
               <height>41</height>
              </rect>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset resource="images/icons.qrc">
               <normaloff>:/images/function_icon_set/add_48.png</normaloff>:/images/function_icon_set/add_48.png</iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>64</width>
               <height>64</height>
              </size>
             </property>
             <property name="autoRaise">
              <bool>true</bool>
             </property>
            </widget>
           </widget>
           <widget class="QWidget" name="layoutWidget">
            <property name="geometry">
             <rect>
              <x>190</x>
              <y>0</y>
              <width>171</width>
              <height>61</height>
             </rect>
            </property>
            <layout class="QGridLayout" name="gridLayout">
             <item row="0" column="0">
              <widget class="QLabel" name="label">
               <property name="font">
                <font>
                 <pointsize>9</pointsize>
                </font>
               </property>
               <property name="text">
                <string>outcome:</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QLabel" name="cur_outcome_lbl">
               <property name="font">
                <font>
                 <pointsize>9</pointsize>
                </font>
               </property>
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="label_3">
               <property name="font">
                <font>
                 <pointsize>9</pointsize>
                </font>
               </property>
               <property name="text">
                <string>follow-up:</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QLabel" name="cur_time_lbl">
               <property name="font">
                <font>
                 <pointsize>9</pointsize>
                </font>
               </property>
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="dataset_file_lbl">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menu_bar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>985</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_file">
    <property name="font">
     <font>
      <family>Verdana</family>
     </font>
    </property>
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="action_save"/>
    <addaction name="action_save_as"/>
    <addaction name="action_open"/>
    <addaction name="action_quit"/>
   </widget>
   <widget class="QMenu" name="menuAnalysis">
    <property name="title">
     <string>Analysis</string>
    </property>
    <widget class="QMenu" name="menuMetric">
     <property name="title">
      <string>metric</string>
     </property>
    </widget>
    <addaction name="menuMetric"/>
    <addaction name="separator"/>
    <addaction name="action_go"/>
    <addaction name="separator"/>
    <addaction name="action_cum_ma"/>
    <addaction name="action_loo_ma"/>
    <addaction name="action_subgroup_ma"/>
    <addaction name="separator"/>
    <addaction name="action_meta_regression"/>
   </widget>
   <widget class="QMenu" name="menuDataset">
    <property name="title">
     <string>Dataset</string>
    </property>
    <addaction name="action_new_dataset"/>
    <addaction name="separator"/>
    <addaction name="action_edit"/>
    <addaction name="action_view_network"/>
    <addaction name="action_add_covariate"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="action_undo"/>
    <addaction name="action_redo"/>
    <addaction name="action_copy"/>
    <addaction name="action_paste"/>
   </widget>
   <addaction name="menu_file"/>
   <addaction name="menuEdit"/>
   <addaction name="menuAnalysis"/>
   <addaction name="menuDataset"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action_save">
   <property name="text">
    <string>save</string>
   </property>
  </action>
  <action name="action_open">
   <property name="text">
    <string>open...</string>
   </property>
  </action>
  <action name="action_quit">
   <property name="text">
    <string>quit</string>
   </property>
  </action>
  <action name="action_go">
   <property name="text">
    <string>meta-analysis...</string>
   </property>
  </action>
  <action name="action_edit">
   <property name="text">
    <string>edit...</string>
   </property>
  </action>
  <action name="action_view_network">
   <property name="text">
    <string>view network...</string>
   </property>
  </action>
  <action name="action_add_covariate">
   <property name="text">
    <string>add covariate...</string>
   </property>
  </action>
  <action name="action_cum_ma">
   <property name="text">
    <string>cumulative meta-analysis...</string>
   </property>
  </action>
  <action name="action_loo_ma">
   <property name="text">
    <string>leave-one-out meta-analysis...</string>
   </property>
  </action>
  <action name="actionOR">
   <property name="text">
    <string>OR</string>
   </property>
  </action>
  <action name="actionRR">
   <property name="text">
    <string>RR</string>
   </property>
  </action>
  <action name="actionTX_Mean_one_arm">
   <property name="text">
    <string>TX Mean (one-arm)</string>
   </property>
  </action>
  <action name="actionYo">
   <property name="text">
    <string>yo</string>
   </property>
  </action>
  <action name="actionNew_dataset">
   <property name="text">
    <string>new dataset...</string>
   </property>
  </action>
  <action name="action_new_dataset">
   <property name="text">
    <string>new dataset...</string>
   </property>
  </action>
  <action name="action_meta_regression">
   <property name="text">
    <string>meta-regression</string>
   </property>
  </action>
  <action name="action_undo">
   <property name="text">
    <string>undo (ctrl + z)</string>
   </property>
  </action>
  <action name="action_redo">
   <property name="text">
    <string>redo (ctrl + y)</string>
   </property>
  </action>
  <action name="action_copy">
   <property name="text">
    <string>copy (ctrl + c)</string>
   </property>
  </action>
  <action name="action_paste">
   <property name="text">
    <string>paste (ctrl + v)</string>
   </property>
  </action>
  <action name="action_subgroup_ma">
   <property name="text">
    <string>subgroup meta-analysis</string>
   </property>
  </action>
  <action name="action_save_as">
   <property name="text">
    <string>save as...</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="images/icons.qrc"/>
 </resources>
 <connections/>
</ui>
